<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QMainWindow" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fenetre</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="rcJoypad"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="actualiser">
         <property name="text">
          <string>Actualiser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="port"/>
       </item>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>deconnecter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOption"/>
    <addaction name="actionSauvegarder_la_configuration"/>
    <addaction name="actionOuvrir_une_configuration"/>
    <addaction name="actionMettre_la_configuration_par_default"/>
    <addaction name="separator"/>
    <addaction name="actionQuiter"/>
   </widget>
   <widget class="QMenu" name="menuaffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="actionMode_1"/>
     <addaction name="actionMode_2"/>
    </widget>
    <addaction name="menuMode"/>
    <addaction name="separator"/>
    <addaction name="actionOuvrir_monitor_Vjoy"/>
   </widget>
   <widget class="QMenu" name="menuCarte">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Carte</string>
    </property>
    <addaction name="actionActualiser"/>
    <addaction name="actionInformations"/>
    <addaction name="separator"/>
    <addaction name="actionOuvrir_le_moniteur_s_rie"/>
    <addaction name="actionD_conecter"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuaffichage"/>
   <addaction name="menuCarte"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSauvegarder_la_configuration">
   <property name="text">
    <string>Sauvegarder la configuration</string>
   </property>
  </action>
  <action name="actionOuvrir_une_configuration">
   <property name="text">
    <string>Ouvrir une configuration</string>
   </property>
  </action>
  <action name="actionMettre_la_configuration_par_default">
   <property name="text">
    <string>Mettre la configuration par default</string>
   </property>
  </action>
  <action name="actionQuiter">
   <property name="text">
    <string>Quiter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMode_1">
   <property name="text">
    <string>Mode 1</string>
   </property>
  </action>
  <action name="actionMode_2">
   <property name="text">
    <string>Mode 2</string>
   </property>
  </action>
  <action name="actionOuvrir_monitor_Vjoy">
   <property name="text">
    <string>Ouvrir monitor Vjoy</string>
   </property>
  </action>
  <action name="actionPort">
   <property name="text">
    <string>Port</string>
   </property>
  </action>
  <action name="actionInformations">
   <property name="text">
    <string>Informations</string>
   </property>
  </action>
  <action name="actionOuvrir_le_moniteur_s_rie">
   <property name="text">
    <string>Ouvrir le moniteur série</string>
   </property>
   <property name="toolTip">
    <string>Ouvre le moniteur série</string>
   </property>
  </action>
  <action name="actionD_conecter">
   <property name="text">
    <string>Déconecter</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionActualiser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
